agent/stats/types.go

```diff
@@ -70,6 +73,7 @@ type StatsContainer struct {
 	client            dockerapi.DockerClient
 	statsQueue        *Queue
 	resolver          resolver.ContainerMetadataResolver
+	config            *config.Config
```

Comment:
config.Config was added to this struct so that we have access to the default stats publishing interval at this scope.